unitOneAssessmentPractice.js

```diff
@@ -4,39 +4,91 @@ let assert = require('assert')
 
 // Write a function called double that doubles a number
 
+const double = (num) => {
+  return num * 2
+}
+
 // Uncomment out the next line to test your solution
-// runQ1Tests()
+runQ1Tests()
 
 // Question Two:
 
 // Write a function called containsSeven that returns whether or not a number has an sevens in it.
 
+const containsSeven = (num) => {
+    let str = num.toString()
+    let seven = "7"
+    if(str.includes(seven)){
+      return true
+    } else {
+      return false
+    }
+  }
+  
+
+
 // Uncomment out the next line to test your solution
-// runQ2Tests()
+runQ2Tests()
 
 // Question Three:
 
 // Write a function called capitalizeTheAs the capitalizes all of the lowercase "a"s in a string leaving all other characters the same
 
+const capitalizeTheAs = str =>{
+  let newStr = '';
+  let letter = '';
```

Comment:
You don't need this variable, you can compare on line 42`if(str[i] === "a")`, here you're comparing "a" to letter, which is an empty string.